riotfile.py

```diff
@@ -145,7 +145,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         ),
         Venv(
             name="tracer",
-            command="pytest {cmdargs} tests/tracer/",
+            command="pytest {cmdargs} tests/tracer/ tests/vendor tests/contrib/logging",
```

Comment:
Could also just move these directories under tests/tracer

Reply:
I think this is ok as-is, since it matches the directory structure in the package.